data-collection-scripts/ltc-scraper/Utah_Data_Extract.py

```diff
@@ -0,0 +1,47 @@
+import json
+from datetime import datetime
+from pytz import timezone
+from urllib.request import urlopen
+
+def load_json():
+    url = ("https://services6.arcgis.com/KaHXE9OkiB9e63uE/ArcGIS/rest/services/COVID19_Long_Term_Care_Facility_Impacts/FeatureServer/273/query?where=1%3D1&objectIds=&time=&geometry=&geometryType=esriGeometryEnvelope&inSR=&spatialRel=esriSpatialRelIntersects&resultType=none&distance=0.0&units=esriSRUnit_Meter&returnGeodetic=false&outFields=*&returnGeometry=true&featureEncoding=esriDefault&multipatchOption=xyFootprint&maxAllowableOffset=&geometryPrecision=&outSR=&datumTransformation=&applyVCSProjection=false&returnIdsOnly=false&returnUniqueIdsOnly=false&returnCountOnly=false&returnExtentOnly=false&returnQueryGeometry=false&returnDistinctValues=false&cacheHint=true&orderByFields=&groupByFieldsForStatistics=&outStatistics=&having=&resultOffset=&resultRecordCount=&returnZ=false&returnM=false&returnExceededLimitFeatures=true&quantizationParameters=&sqlFormat=none&f=pjson&token=")
+    response = urlopen(url)
+    if response.getcode() == 200:
+        source = response.read()
+        dict_data = json.loads(source)
+    else:
+        print("Error")
```

Comment:
If this fails for any reason, let's print an error message with the error code and then `sys.exit(1)` or some other non-zero return code so that the script terminates and automation knows the script failed and can report the error